src/lib/DocSearch.js

```diff
@@ -184,6 +190,17 @@ class DocSearch {
     input.setVal('');
     window.location.href = suggestion.url;
   }
+
+  handleKeyboardShortcuts(input) {
+    $(document).keydown(function(e) {
+      if (false === input.is(':focus') && (e.which == 191 || e.which == 83)) {
```

Comment:
> I think you might fallback to e.keyCode when e.which is not available. I kind of remember IE not following the which spec here. cc @vvo

`which` is OK since IE 9 (http://caniuse.com/#search=which)